enterprise/server/backends/redis_client/redis_client.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var (
-	defaultRedisTarget          = flag.String("app.default_redis_target", "", "A Redis target for storing remote shared state. To ease migration, the redis target from the remote execution config will be used if this value is not specified.")
+	defaultRedisTarget          = flag.String("app.default_redis_target", "localhost:6379", "A Redis target for storing remote shared state. To ease migration, the redis target from the remote execution config will be used if this value is not specified.")
```

Comment:
This might be nice for local development, but for most deployments it's going to be something else and I think we'd rather show them an error message that it's not set.